pyproject.toml

```diff
@@ -0,0 +1,74 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+# Check https://flit.readthedocs.io/en/latest/pyproject_toml.html for all available sections
+name = "pyansys"
+version = "2023.1.dev0"
+description = "Pythonic interfaces to Ansys products"
+readme = "README.rst"
+license = "MIT"
+homepage = "https://github.com/pyansys"
+repository = "https://github.com/pyansys/pyansys_landing"
+documentation = "https://docs.pyansys.com/"
+authors = [
+    "ANSYS, Inc. <pyansys.support@ansys.com>",
+]
+maintainers = [
+    "PyAnsys developers <pyansys.maintainers@ansys.com>",
+]
+classifiers = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Science/Research',
+    'Topic :: Scientific/Engineering :: Information Analysis',
+    'License :: OSI Approved :: MIT License',
+    'Operating System :: Microsoft :: Windows',
+    'Operating System :: POSIX',
+    'Operating System :: MacOS',
+    'Programming Language :: Python :: 3.7',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',

```

Comment:
```suggestion
    'Programming Language :: Python :: 3.9',
    'Programming Language :: Python :: 3.10',
```

Reply:
I honestly have no idea why the text is all red.

Reply:
Sounds weird. I don't see anything in red. I'll look into it anyway

![image](https://user-images.githubusercontent.com/37798125/182774276-5e71aeb5-660c-49d3-88c4-efcfe2122709.png)